osu.Game/Overlays/BeatmapSetOverlay.cs

```diff
@@ -66,10 +96,17 @@ public BeatmapSetOverlay()
                     RelativeSizeAxes = Axes.Both,
                     Colour = OsuColour.Gray(0.2f)
                 },
+                loading = new LoadingAnimation
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Alpha = 1,

```

Comment:
because of the way LoadingAnimation is implemented, i recommend you use `State = States.Visible` here (and replace the `FadeIn`/`FadeOut` calls with `Show`/`Hide`.